changelogs/fragments/9651-iocage-inventory-hooks.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - iocage inventory plugin - the new parameter ``hooks`` of the plugin is a list of files inside a jail that provide configuration parameters for the inventory. The inventory plugin reads the files from the jails and put the contents into the items of created variable ``iocage_hooks`` (https://github.com/ansible-collections/community.general/issues/9650, https://github.com/ansible-collections/community.general/pull/9651).
```

I think there's one thing left (sorry I didn't notice earlier), then it's done:
```suggestion
  - iocage inventory plugin - the new parameter ``hooks_results`` of the plugin is a list of files inside a jail that provide configuration parameters for the inventory. The inventory plugin reads the files from the jails and put the contents into the items of created variable ``iocage_hooks`` (https://github.com/ansible-collections/community.general/issues/9650, https://github.com/ansible-collections/community.general/pull/9651).
```

---

My fault, actually.  Thank you.